CHANGES.rst

```diff
@@ -15,7 +15,9 @@ Improvements
 Bugfixes
 ^^^^^^^^
 
-- None so far :)
+- Users cannot submit multiple anonymous surveys anymore by logging out and in again.
```

Comment:
I think most of the other "shorter" entries don't have a period at the end ;)

Reply:
Is it a bugfix or an improvement? Not sure tbh... one could say it was expected behavior, we just improved it, or one could say that it should never have allowed those submissions...

Reply:
Agree, this fits more as an improvement.